README.md

```diff
@@ -1 +1,16 @@
-# game
\ No newline at end of file
+# Orion's Belt BattleGrounds
+[![Build Status](https://travis-ci.org/orionsbelt-battlegrounds/game.svg?branch=master)](https://travis-ci.org/orionsbelt-battlegrounds/game)
+
+### Running
+
+You need to install [lein](http://leiningen.org/).
+
+* `lein server` - starts the server
+* `lein frontend` - starts the frontend, in development mode
+
+After both processes started, go to [http://localhost:3450/dev.html](http://localhost:3450/dev.html).
+
+### Tests
+
+* `lein test` - runs the full test suite
+* `lein autotest` - listen for file changes and is allways running tests
```

Comment:
Where does this `lein` command come from? Didn't you mean `lein test-refresh`? This is probably something that I just don't know...

Oh, and you have a typo (allways -> always).